samples/ar-view-node/src/main/java/io/github/sceneview/sample/arviewnode/FrameTime.kt

```diff
@@ -0,0 +1,32 @@
+package io.github.sceneview.sample.arviewnode
+
+import kotlin.time.Duration
+import kotlin.time.Duration.Companion.nanoseconds
+import kotlin.time.DurationUnit
+
+data class FrameTime(val nanoseconds: Long, val lastNanoseconds: Long? = null) {
+
+    /**
+     * ### The duration between this frame and the last frame
+     */
+    val interval: Duration by lazy { interval(lastNanoseconds) }
+
+    val intervalSeconds: Double by lazy { intervalSeconds(lastNanoseconds) }
+
+    val fps: Double by lazy { fps(lastNanoseconds) }
+
+    /**
+     * ### The duration between this frame and the last frame
+     */
+    private fun interval(lastNanoseconds: Long?): Duration =
+        (nanoseconds - (lastNanoseconds ?: 0)).nanoseconds
+
+    private fun intervalSeconds(lastNanoseconds: Long?): Double = interval(lastNanoseconds).toDouble(
+        DurationUnit.SECONDS
+    )
+
+    private fun fps(lastNanoseconds: Long?): Double = 1.0 / intervalSeconds(lastNanoseconds)
+
+    fun fps(frameTime: FrameTime?): Double = fps(frameTime?.nanoseconds)
```

Comment:
Is this method correct and still used?

Reply:
I can remove this. Currently it is part of my TrackingNode implementation, to find out if the current FPS value is below a certain FPS values based on a Threshold. But for this sample, it isn't really important.

Reply:
So does it help to calculate FPS between two frames if `lastNanoseconds` isn't available?

Reply:
Not really, the truth is, that the TrackingNode is more smooth if I remove the fps calculation :).